apps/wing/src/commands/init.ts

```diff
@@ -99,7 +99,7 @@ export async function init(template: string, options: InitOptions = {}): Promise
   options.template = template;
   // Check if the template exists for the selected language
   const templatePath = join(PROJECT_TEMPLATES_DIR, language, template);
-  const templateExists = await exists(templatePath);
+  const templateExists = await exists(templatePath, constants.F_OK | constants.R_OK); // eslint-disable-line no-bitwise
```

Comment:
Is just `R_OK` sufficient? I thought R_OK is both "it exists and is readable", but worth checking that